global.css

```diff
@@ -23,7 +23,7 @@ body {
 .logo {
   cursor: pointer;
   vertical-align: top;
-  height: 100%;
+  height: 4rem;
```

Comment:
the 4rem height forces the main logo to stretch on mobile. was the previous 100% height not OK for accessibility issues?
![Screen Shot 2020-08-03 at 12 11 17 PM](https://user-images.githubusercontent.com/50561164/89203523-b22d4f00-d582-11ea-83ed-91fb4513ad44.png)

Reply:
Ew changed it back to 100%, still stretched for me though?

![image](https://user-images.githubusercontent.com/35791997/89234455-0273d380-d5ba-11ea-9a41-dc62fd1d8812.png)

The 4rem was meant just for the logo in the nav to stop it from getting to big, just fixed that so it will only change that one now.